instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/MapboxHistoryTest.kt

```diff
@@ -354,6 +356,19 @@ class MapboxHistoryTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.j
         assertEquals(thirdJson, thirdFile.findCustomEvent().properties)
     }
 
+    @Test
+    fun trip_session_resets_successfully() {
```

Comment:
Why is it in `MapboxHistoryTest`? Is this related?

Reply:
We typically use this function with history and replayer, I can create a new test class though, I was on the fence about that.